### ISSUE TITLE
Ups, you mean "dns_blocklists.**cf**" right?

### ISSUE THREAD

**Issue by bh** [mailcow/mailcow-dockerized#6174]
Ups, you mean "dns_blocklists.**cf**" right?

``` sh
$ find . -name dns_blocklists.cnf
$ find . -name dns_blocklists.cf
./data/conf/postfix/dns_blocklists.cf
```

_Originally posted by @bh in https://github.com/mailcow/mailcow-dockerized/pull/6160#discussion_r1844023870_

**Comment by DerLinkman** [comment]
Yes, i'm sorry.

**Comment by DerLinkman** [comment]
Fixed with commit: 5ad4ab5b60b5f4496d01b623dca44d862cfff868

**Comment by bh** [comment]
Super, danke ✌️
